.github/workflows/publishing.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish new release on PyPI
+
+# workflow is triggered when a new release is created or a new git tag is pushed to this repository
+# and it can be manually triggered
+on:
+  workflow_dispatch:
+  release:
+    types: [created]
+  push:
+    tags:
+      - '*'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    # downloads the repository code to the runner's file system for workflow access
+    - uses: actions/checkout@v2
+    - name: Fetch all history for all tags and branches
+      run: git fetch --prune --unshallow
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.6'
```

Comment:
Thank you for mentioning that. It should be Version 3.8, I will change it. I thought that by building it with our minimal version, it will show the backwards compatibility of our project.